### PR TITLE
docs(ci.md): added google cloud CI docs for NodeJs using the playwright Docker Image

### LINKED ISSUE #27769: Add Google Cloud Build to CI Docs
Hi,

I have integrated Playwright using Google Cloud Build and NodeJS and I want to update the docs: https://github.com/microsoft/playwright/blob/main/docs/src/ci.md to add in a Google Cloud Build section so people can also use Google Cloud Build to run Playwright tests.

I read the guide asking to raise an issue before starting any work. Is that OK for me to do?

### PR DESCRIPTION
**Pull request by hemang-rajyaguru**
Fixes #27769

### CODE REVIEW

**Comment by hemang-rajyaguru** [comment]
Any chance for a review @yury-s ? Thanks 👍🏾

File: docs/src/ci.md [hunk 1]
```diff
@@ -587,6 +587,20 @@ tests:
     - npm ci
     - npx playwright test --project=$PROJECT --shard=$SHARD
 ```
+### Google Cloud Build
+
+To run Playwright tests on Google Cloud Build, use our public Docker image ([see Dockerfile](./docker.md)).
+
+```yml
+steps:
+- name: 'mcr.microsoft.com/playwright:v%%VERSION%%-jammy'
```
Comment (yury-s) [inline docs/src/ci.md:596]: we don't use quotes around image name in other places, let's be consistent

File: docs/src/ci.md [hunk 2]
```diff
@@ -587,6 +587,20 @@ tests:
...
+  script: |
+    #!/usr/bin/env bash
```
Comment (yury-s) [inline docs/src/ci.md:598]: This doesn't help much the users, let's replace it with `...` as in GitLab CI?

File: docs/src/ci.md (hunk 1 above)
Comment (hemang-rajyaguru) [reply docs/src/ci.md]: Done

File: docs/src/ci.md (hunk 2 above)
Comment (hemang-rajyaguru) [reply docs/src/ci.md]: Done

File: docs/src/ci.md [hunk 3]
```diff
@@ -587,6 +587,18 @@ tests:
     - npm ci
     - npx playwright test --project=$PROJECT --shard=$SHARD
 ```
+### Google Cloud Build
+
```
Comment (mxschmitt) [inline docs/src/ci.md:591]:
```suggestion
* langs: js

```

Either make it JS only or add snippets for all the supported languages.

Comment (hemang-rajyaguru) [reply docs/src/ci.md:592]: Done

**Comment by hemang-rajyaguru** [comment]
Let me know if there is anything else to add @yury-s @mxschmitt 👍

File: docs/src/ci.md [hunk 4]
```diff
@@ -587,6 +587,19 @@ tests:
...
+To run Playwright tests on Google Cloud Build, use our public Docker image ([see Dockerfile](./docker.md)).
+
+```yml
+steps:
+- name: mcr.microsoft.com/playwright:v%%VERSION%%-jammy
+  script: 
+  ...
+  env:
+  - 'CI=true'
```
Comment (yury-s) [inline docs/src/ci.md:601]: Does Google Clould CI not set it by default? I thought all modern cloud CI providers run with `CI=1`, we don't have it any other example.

Comment (hemang-rajyaguru) [reply docs/src/ci.md:601]: From what I have experienced it does not set it by default. I had to set it to trigger the CI rules in playwright config

**Review by yury-s — APPROVED** [review]

**Pull request merged**